antropy/fractal.py

```diff
@@ -194,7 +194,7 @@ def katz_fd(x, axis=-1):
     return kfd
 
 
-@jit((types.Array(types.float64, 1, "C", readonly=True), types.int32))
+@jit((types.Array(types.float64, 1, "C", readonly=True), types.int32), nopython=True)
```

Comment:
Was the only `@jit` decorated function that did not have the `nopython` flag set to True. After quickly scanning the numba docs I believe it should be save (& faster) to set this to `True`.

Benchmarking shows no impact at all :arrow_down: 


flag not set:
```py
In [1]: import numpy as np; import antropy as ant

In [2]: arr = np.random.randn(10_000).astype(np.float64)

In [3]: %timeit ant.higuchi_fd(arr)
268 µs ± 14.8 µs per loop (mean ± std. dev. of 7 runs, 1,000 loops each)
```

flag set:
```py
In [1]: import numpy as np; import antropy as ant

In [2]: arr = np.random.randn(10_000).astype(np.float64)

In [3]: %timeit ant.higuchi_fd(arr)
269 µs ± 15.3 µs per loop (mean ± std. dev. of 7 runs, 1,000 loops each)
```

Reply:
Awesome thanks!